### ISSUE TITLE
HTTP Request Status Code 410 (Gone)

### ISSUE THREAD

**Issue by mainframed** [skilion/onedrive#113]
Looks like this doesn't work anymore?

```
 onedrive -v
Loading config ...
Initializing the OneDrive API ...
Opening the item database ...
All operations will be performed in: /home/x/Documents/OneDrive
Initializing the Synchronization Engine ...
Applying differences ...
HTTP request returned status code 410 (Gone)
Applying differences ...
HTTP request returned status code 410 (Gone)
Applying differences ...
sync.SyncException@src/sync.d(106): HTTP request returned status code 410 (Gone)
----------------
??:? void sync.SyncEngine.applyDifferences() [0x59c3e4]
??:? void main.performSync(sync.SyncEngine) [0x58c58b]
??:? _Dmain [0x58bdce]
??:? _D2rt6dmain211_d_run_mainUiPPaPUAAaZiZ6runAllMFZ9__lambda1MFZv [0x5a966e]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate()) [0x5a95b8]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).runAll() [0x5a962a]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate()) [0x5a95b8]
??:? _d_run_main [0x5a9529]
??:? main [0x58f5d5]
??:? __libc_start_main [0x5ce1682f]
onedrive.OneDriveException@src/onedrive.d(348): HTTP request returned status code 410 (Gone)
----------------
??:? std.json.JSONValue onedrive.OneDriveApi.get(const(char)[]) [0x593c62]
??:? void sync.SyncEngine.applyDifferences() [0x59c167]
??:? void main.performSync(sync.SyncEngine) [0x58c58b]
??:? _Dmain [0x58bdce]
??:? _D2rt6dmain211_d_run_mainUiPPaPUAAaZiZ6runAllMFZ9__lambda1MFZv [0x5a966e]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate()) [0x5a95b8]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).runAll() [0x5a962a]
??:? void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate()) [0x5a95b8]
??:? _d_run_main [0x5a9529]
??:? main [0x58f5d5]
??:? __libc_start_main [0x5ce1682f]
[2]    27810 segmentation fault (core dumped)  onedrive -v
```

**Comment by wm8120** [comment]
Same problem found here.

**Comment by skilion** [comment]
Solved in 7257c4c9bfb07efe4e333f562868e2ad80b739d3
